examples/dax/cluster.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: dax.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: example-dax-cluster
```

Comment:
think for e2e tests it would be better to use only the ref variant - what do you think ?

Reply:
I agree, I have removed the non-ref variant in [`0aac4`](https://github.com/crossplane/provider-aws/pull/1256/commits/0aac42d58a30c4e20b55d6810dc00363ba1ab2c2).